rplugin/python/nvim_ipy.py

```diff
@@ -265,7 +269,7 @@ def ipy_run(self, args):
             if p.get("source") == "page":
                 # TODO: if this is long, open separate window
                 self.append_outbuf(p['text'])
-
+        self.buf.name = self.base_buf_name + '({}s elapsed)'.format(time.time()-start_time) 
```

Comment:
I think that it should be possible to have execution time in there. Specifically in a REPL setting would allow to quickly find identify the hotspots in the code by running code line by line without having to modify the code.  But maybe all that is needed is to be able to "decorate" each call to the kernel and display the output somewhere. This way it would be possible to wrap each command with code to compute a timedelta.